### PR TITLE
Upgrade cliquet and add init command to make serve

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
Do we want to rename the command first ?

**Comment by leplatrem** [comment]
@ametaireau @Natim r?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,7 +7,17 @@ This document describes changes between each past release.
 0.2.2 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Upgraded to *cliquet* 1.6.0
```
Comment (leplatrem) [inline CHANGELOG.rst]: 1.7.0

File: .travis.yml [hunk 2]
```diff
@@ -18,3 +18,7 @@ install:
 script:
     - if [[ $ACTION != loadtest ]]; then tox -e $TOX_ENV; fi
     - if [[ $ACTION == loadtest ]]; then make loadtest-check; fi
+after_success:
+    # Report coverage results to coveralls.io
+    - pip install coveralls
+    - coveralls
```
Comment (Natim) [inline .travis.yml]: Missing file end line

**Pull request merged**